src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDriverAdapter.java

```diff
@@ -4,17 +4,15 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
-import edu.kis.powp.jobs2d.features.DrawerFeature;
-import edu.kis.powp.jobs2d.features.DriverFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class DrawPanelAdapter implements Job2dDriver {
+public class Job2dDriverAdapter implements Job2dDriver {
```

Comment:
Now the name does not indicate the purpose of the adapter or what was actually adapted.